akka-projection-cassandra/src/test/scala/akka/projection/cassandra/ContainerSessionProvider.scala

```diff
@@ -41,7 +42,12 @@ object ContainerSessionProvider {
     if (disabled)
       Future.successful(())
     else
-      Future.fromTry(Try(container.start()))
+      Future.fromTry(Try {
+        container.underlyingUnsafeContainer
+          .withStartupCheckStrategy(new IsRunningStartupCheckStrategy)
+          .withStartupAttempts(5)
+        container.start()
+      })
```

Comment:
I don't see a reason why we are wrapping it on a future. Maybe to be able to put a timeout on it? 

The start method only returns when the container is ready or failed to start. Then, in the beforeAll method we wait up to 30 seconds for it to complete.

Reply:
Yes, I think the timeout is the only reason.

Reply:
Ok, I will do the same for the JDBC one, but unfortunately we can't do for the Slick one. 

The Slick is special because we must start the container before we can create DB settings because we need to know the URL.